database/factories/FreeleechTokenFactory.php

```diff
@@ -6,5 +6,7 @@
 
 $factory->define(App\Models\FreeleechToken::class, function (Faker $faker) {
     return [
+        'user_id'    => $faker->randomNumber(),
+        'torrent_id' => $faker->randomNumber(),
```

Comment:
It would be ideal to enforce foreign key constraints here. I'm curious why the automated process used `randomNumber()` here. I'll check later, but for now, I'm noting that these should really honor FKs constraints. I would expect something more like what we see elsewhere in similar usages:

```php
        'user_id' => function () {
            return factory(App\Models\User::class)->create()->id;
        },
        'torrent_id' => function () {
            return factory(App\Models\Torrent::class)->create()->id;
        },
```